packages/experience/src/hooks/use-fallback-route.ts

```diff
@@ -0,0 +1,13 @@
+import { experience } from '@logto/schemas';
+import { useMemo } from 'react';
+
+const useFallbackRoute = () =>
+  useMemo(() => {
+    const fallbackKey = new URLSearchParams(window.location.search).get('fallback');
```

Comment:
Add the `fallback` search parameter to the `utils/search-parameters.ts`, avoid using the plain text key.